specs/trip_dispatch_spec.rb

```diff
@@ -88,5 +89,144 @@
       passenger.must_be_instance_of RideShare::Passenger
       passenger.trips.must_include trip
     end
+
+    it "stores start_time as instances of Time" do
+      dispatcher = RideShare::TripDispatcher.new
+
+      all_start_times = dispatcher.trips.all? { |trips| trips.start_time.class == Time }
+
+      all_start_times.must_equal true
+
+
+    end
+
+    it "stores end_time as instances of Time" do
+      dispatcher = RideShare::TripDispatcher.new
+
+      all_end_times = dispatcher.trips.all? { |trips| trips.end_time.class == Time }
+
+      all_end_times.must_equal true
+    end
+  end
+
+  describe "#assign_driver" do
```

Comment:
I am unconvinced that `assign_driver` needs to be a public method.  Why not make it `private`?